todo_app/app.py

```diff
@@ -1,15 +1,32 @@
-from flask import Flask
-
+from flask import Flask, render_template, redirect, url_for, request
+from flask.globals import session
+from todo_app.data.session_items import add_item, get_items
 from todo_app.flask_config import Config
+from todo_app.trello_items import Get_items_from_trello, create_card,move_card
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
-def index():
-    return 'Hello World!'
+def index(): 
+    items = Get_items_from_trello()
```

Comment:
Here we're still using the default objects (dictionaries) returned by Trello - we should update either here or in the `get_items_from_trello` method to use a specific `Item` class as per Part 5